### PR TITLE
Increased StandardMethodCodec's WriteBuffer start capacity

### PR DESCRIPTION
**Pull request by gaaclarke**
related: https://github.com/flutter/flutter/pull/101790

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by gaaclarke** [comment]
I'm not going to bother landing the benchmarks first.  The savings are going to be similar to what we had with the other benchmarks.  Although we'll probably benefit even more since method payloads are always going to be bigger.

File: packages/flutter/lib/src/services/message_codecs.dart [hunk 1]
```diff
@@ -9,6 +9,8 @@ import 'package:flutter/foundation.dart' show ReadBuffer, WriteBuffer;
 
 import 'message_codec.dart';
 
+int _writeBufferStartCapacity = 64;
```
Comment (jonahwilliams) [inline packages/flutter/lib/src/services/message_codecs.dart:12]: `const`?

**Review by jonahwilliams — APPROVED** [review]
LGTM with nit

File: packages/flutter/lib/src/services/message_codecs.dart (hunk 1 above)
Comment (gaaclarke) [reply packages/flutter/lib/src/services/message_codecs.dart:12]: oops, thanks.

**Pull request merged**